lewis/adapters/epics.py

```diff
@@ -205,12 +206,17 @@ def start_server(self):
 
             The server does not process requests unless :meth:`handle` is called regularly.
         """
-        self._server = SimpleServer()
-        self._server.createPV(prefix=self._options.prefix,
-                              pvdb={k: v.config for k, v in self.pvs.items()})
-        self._driver = PropertyExposingDriver(target=self, pv_dict=self.pvs)
+        if self._server is None:
+            self._server = SimpleServer()
+            self._server.createPV(prefix=self._options.prefix,
+                                  pvdb={k: v.config for k, v in self.pvs.items()})
+            self._driver = PropertyExposingDriver(target=self, pv_dict=self.pvs)
+
+            self._last_update = datetime.now()
 
-        self._last_update = datetime.now()
+    def stop_server(self):
+        self._server = None
+        self._driver = None
```

Comment:
With the one forced update after starting it seems to work as expected. I also changed the order of "destruction" - although I'm not sure whether that changes anything either.